index.js

```diff
@@ -97,15 +141,17 @@ module.exports = class View extends EventEmitter {
         return this
     }
 
+    slurp = { attr: 'data-js', view: 'data-view', name: 'data-name', img: 'data-src' }
+
     postRender() { return this }
 
     render() {
         if( this.data ) this.model = Object.create( this.Model, { } ).constructor( this.data )
 
-        this.slurp( {
+        this.slurpTemplate( {
             insertion: this.insertion || { el: document.body },
             isView: true,
-            template: Reflect.apply( this.template, this.TemplateContext, [ this.getTemplateOptions() ] )
+            template: Reflect.apply( this.template, this, [ this.getTemplateOptions() ] )
```

Comment:
What's going on with TemplateContext?

Reply:
I think this is a bit too much for right now, but if you're going to move forward with this, then drop Reflect.apply altogether.

template: this.template(this.getTemplateOptions())

Reply:
I'll just save it for the future then and keep `TemplateContext` for now.